lib/i18n/american-english.js

```diff
@@ -270,6 +270,16 @@ class EnglishLanguagePack extends DefaultLanguagePack {
         return [inflected, ...words.slice(1)].join(' ');
     }
 
+    // to verb base
+    toVB(verb) {
```

Comment:
please use long names for the methods, and also please use jsdoc syntax for documentation

Reply:
fixed in d35c0d1